### PR TITLE
feature/VDX-294

### PR DESCRIPTION
**Pull request by sksadjad**
changed the text based on VDX-291 branch

### CODE REVIEW

File: packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json [hunk 1]
```diff
@@ -234,7 +261,12 @@
         {
           "id": "issueSuccess",
           "operation": "navigate",
-          "path": "https://banqup.demo.sphereon.com"
+          "path": "/credentials/issue/success"
+        },
+        {
+          "id": "toCompanyFlow",
```
Comment (BtencateSphereon) [inline packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json]: does not go to the company flow. this goes to the account flow

File: packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json [hunk 2]
```diff
@@ -189,6 +192,30 @@
...
+      "logo": {
+        "src": "banqup/logo.svg"
+      },
+      "rightPaneTitle": "credentials_success_right_pane_valid_id_title",
+      "rightPaneParagraph": "credentials_success_right_pane_registered_company_paragraph",
+      "rightPaneButtonCaption": "label_finish",
+      "rightPaneTextHeight": "40%",
+      "rightPaneTextMarginTop": "0",
+      "rightPaneTextMarginBottom": "10%",
+      "rightPaneButtonWidth": "135px",
```
Comment (BtencateSphereon) [inline packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json:208]: if it is a small fix, can we start using numbers only for widths and heights? i mentioned this on another pr as well. i think we use these only for style props. and react handles the units for you

**Review by BtencateSphereon — CHANGES_REQUESTED** [review]

**Review by BtencateSphereon — CHANGES_REQUESTED** [review]
small remark

File: packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json (hunk 1 above)
Comment (sksadjad) [reply packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json]: changed it to `toAccountFlow`

File: packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json [hunk 3]
```diff
@@ -234,7 +261,12 @@
         {
           "id": "issueSuccess",
           "operation": "navigate",
-          "path": "https://banqup.demo.sphereon.com"
+          "path": "/credentials/issue/success"
```
Comment (BtencateSphereon) [inline packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json]: add a  nextId

File: packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json [hunk 4]
```diff
@@ -1,8 +1,11 @@
 {
   "general": {
-    "credentialName": "Legal entity credential",
+    "credentialName": "Legal Entity",
     "oid4vpAgentBaseUrl": "https://agent.banqup.demo.sphereon.com",
-    "oid4vciAgentBaseUrl": "https://agent.banqup.demo.sphereon.com"
+    "oid4vciAgentBaseUrl": "https://agent.banqup.demo.sphereon.com",
+    "downloadUrl": "https://agent.banqup.demo.sphereon.com/#/download",
+    "verifierUrl": "https://legal-entity.banqup.demo.sphereon.com",
+    "verifierUrlCaption": "Identity credential"
```
Comment (BtencateSphereon) [inline packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json]: should this not be something for legal entity?

Comment (sksadjad) [reply packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json]: changed to `Legal Entity credential`

**Review by BtencateSphereon — APPROVED** [review]

**Pull request merged**